core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -1982,12 +1982,7 @@ public AutoCompletionCandidates doAutoCompleteLabel(@QueryParameter String value
         return Items.findNearest(AbstractProject.class, name, context);
     }
 
-    private static final Comparator<Integer> REVERSE_INTEGER_COMPARATOR = new Comparator<Integer>() {
-        @Override
-        public int compare(Integer o1, Integer o2) {
-            return o2-o1;
-        }
-    };
+    private static final Comparator<Integer> REVERSE_INTEGER_COMPARATOR = (o1, o2) -> o2 - o1;
```

Comment:
ditto

Reply:
This case seems to be fine like this, due to the reverse order, it will be more wordy to call methods.

```
Comparator.comparingInt(o -> o).reversed() // not accepted due to reversed generic being Object
```

Reply:
Sure, I have no strong opinion either way.